lib/http.js

```diff
@@ -1344,6 +1344,22 @@ Agent.prototype.removeSocket = function(s, name, host, port, localAddress) {
 var globalAgent = new Agent();
 exports.globalAgent = globalAgent;
 
+function check_is_token(val) {
```

Comment:
I agree we want to change the name, and I think general we're more `camelCase` style -- perhaps `checkValidToken`